ccpp/driver/GFS_diagnostics.F90

```diff
@@ -2186,6 +2186,61 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
       enddo
     endif
 
+    idx = idx + 1
+    ExtDiag(idx)%axes = 3
+    ExtDiag(idx)%name = 'spp_wts_pbl'
+    ExtDiag(idx)%desc = 'spp pbl perturbation wts'
+    ExtDiag(idx)%unit = 'm/s'
+    ExtDiag(idx)%mod_name = 'gfs_phys'
+    allocate (ExtDiag(idx)%data(nblks))
+    do nb = 1,nblks
+      ExtDiag(idx)%data(nb)%var3 => IntDiag(nb)%spp_wts_pbl(:,:)
```

Comment:
We should be able to just point to the `Coupling` variables here.

Reply:
Removed the Intdiag variable references.